analyzers/src/SonarAnalyzer.CSharp/SymbolicExecution/Common/Constraints/ByteArraySymbolicValueConstraint.cs

```diff
@@ -18,7 +18,7 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-namespace SonarAnalyzer.SymbolicExecution.Common.Constraints
+namespace SonarAnalyzer.SymbolicExecution.Constraints
```

Comment:
I am a bit confused - shouldn't this change be followed by moving these Constraints to a different folder - `analyzers/src/SonarAnalyzer.CSharp/SymbolicExecution/Constraints` ?

Reply:
nevermind, this is done in #5087